bundle/manifests/numaresources-operator.clusterserviceversion.yaml

```diff
@@ -257,9 +264,6 @@ spec:
                   initialDelaySeconds: 5
                   periodSeconds: 10
                 resources:
-                  limits:
-                    cpu: 200m
-                    memory: 100Mi
```

Comment:
I like this, the operator pod should really not have limits until/unless we have a good rationale - and proof the limits will not backfire. And since AFAIK we don't have this data, better not to guess. The only problem is, this should happen more explicitely :\

Reply:
This come from here: https://github.com/openshift-kni/numaresources-operator/pull/71
Was generated after make bundle

Reply:
yup. For the reasons above, it's good we remove the limits. But we should remove the limitis explicitely, with a separate commit/PR and proper explanation, and not causally by incident on an apparently unrelated PR which happened to need to run `make bundle`.

Reply:
Ok, done: https://github.com/openshift-kni/numaresources-operator/pull/77

Reply:
Rebase and fix it